openapi/paths/subscriptions.yaml

```
@@ -3,14 +3,14 @@ get:
     - Core
   tags:
     - Orders
-  summary: Retrieve orders
+  summary: Retrieve subscription orders
```

```suggestion
  summary: Retrieve orders
```

This applies for both order types